### PR TITLE
Update UG

### PR DESCRIPTION
**Pull request by taggyhan**
- Checking for bugs

### CODE REVIEW

**Comment by taggyhan** [comment]
fix #138

**Comment by taggyhan** [comment]
fix bug on redo in UG

**Review by starchypotatocode — APPROVED** [review]

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -158,7 +158,7 @@ The `Model` component,
 
 The `Storage` component,
 
-- can save both hall pointer data and user preference data in JSON format, and read them back into corresponding objects.
+- can save both hall pointer data and user preference data in JSON format, and read them back into scorresponding objects.
```
Comment (yuvrajaryan) [inline docs/DeveloperGuide.md]: typo

Comment (taggyhan) [reply docs/DeveloperGuide.md]: ok fixed

**Comment by keithxun** [comment]
lgtm

**Pull request merged**